types/badgen/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for badgen 2.6
+// Project: https://github.com/amio/badgen
+// Definitions by: m5d215 <https://github.com/m5d215>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export type ColorPresets =
+    | 'green'
+    | 'blue'
+    | 'red'
+    | 'yellow'
+    | 'orange'
+    | 'purple'
+    | 'pink'
+    | 'grey'
+    | 'gray'
+    | 'cyan'
+    | 'black';
+
+export interface BadgeCreationParams {
+    /**
+     * Text representing the subject.
+     */
+    subject: string;
+
+    /**
+     * Text representing the status.
+     */
+    status: string;
+
+    /**
+     * Color RGB or Color Name, optional.
+     */
+    color?: ColorPresets | string;
+
+    /**
+     * 'flat' or undefined, optional.
+     */
+    style?: 'flat';
+
+    /**
+     * Use icon, optional.
+     */
+    icon?: string;
+
+    /**
+     * Use this if icon is not square.
+     * @default 13
+     */
+    iconWidth?: number;
+}
+
+export default function badgen(params: BadgeCreationParams): string;
```

Comment:
It seems like the [underlying library](https://unpkg.com/badgen@2.6.0/lib/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.